installers/ansible/inventory

```diff
@@ -210,8 +210,10 @@ db_user='testuser'
 # Operator defaults are listed with each variable
 #
 # "default_instance_memory" represents the memory request for a PostgreSQL
-# instance
-# default_instance_memory='512Mi'
+# instance. For production instances, this should be at least 512Mi, but we
+# model a number that is OK for just basic deployment testing. However, you
+# really should make this value bigger
+# default_instance_memory='128Mi'
```

Comment:
Should this be uncommented? If not the default value will be different for the ansible install and the installer. Because it is uncommented in the template, and the value in `postgres-operator.yml` is set.

Reply:
Yes, you're corect.

Reply:
cool, other than that lgtm. Tested it out with the ansible install